components/brave_wallet/browser/zcash/rust/authorized_orchard_bundle.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_ZCASH_RUST_AUTHORIZED_ORCHARD_BUNDLE_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_ZCASH_RUST_AUTHORIZED_ORCHARD_BUNDLE_H_
+
+#include <vector>
+
+namespace brave_wallet::orchard {
+
+class AuthorizedOrchardBundle {
```

Comment:
what is an authorized vs unauthorized orchard bundle? It seems odd to me that they have identical public api, but are two distinct classes that don't share an interface

Reply:
Added comments